supplementary/alica_ros1/alica_ros_turtlesim/libalica-turtlesim/src/GoTo.cpp

```diff
@@ -18,7 +18,8 @@ void GoTo::run()
 {
     // solve constraints and get value
     if (!_query.getSolution<alica::reasoner::CGSolver, double>(getPlanContext(), _results)) {
-        alica::Logging::logError(alica::LOGNAME) << "Behaviour: " << getName() << ", solution to query not found";
+        alica::Logging::logWarn(alica::LOGNAME) << "Behaviour: " << getName() << " calculating ...";
+        return;
```

Comment:
is there a risk of calculating forever?

Reply:
If someone wants tor run with 999999999 turtles I hope they are ready for that risk

Reply:
I checked here, by the behavior itself it can run forever, and there's no timeout (do we have a timeout mechanism built in behaviors or states where I can add this parameter in plan designer?).

`GSolver` logic implements a timeout that comes from "Alica.CSPSolving.MaxSolveTime" set in `Alica.yaml`, but this timeout is just for a simple pass in the gradient method, and if I understand correctly once the 10ms (this is the timeout set right now) are passed it will return what it has found and somehow it will use this result as a seed for the next tick of run() (is this correct? I could not find the portion of code that sets the seeds exactly, but the behavior points that this is happening).

Therefore, `GoTo` method needs a timeout implemented in order to be correct. However, this application will never timeout for a reasonable quantity of robots because of the nature of it (just simple paths being calculated in a 2D environment without collision by a holonomic robot).

If you want I can implement it, is it necessary here?